core/internal/testdata/new_round_log.json

```diff
@@ -10,9 +10,9 @@
       "blockHash": "0x5e3bd2cc97a68136cead922330e2ec27201420b3eff182875e388474079fcd9e",
       "blockNumber": "0xa",
       "address": "0x2fCeA879fDC9FE5e90394faf0CA644a1749d0ad6",
-      "data": "0x",
+      "data": "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
We updated `NewRound` to have an extra non-`indexed` field, but never updated the test fixture.

Reply:
A few weeks ago, iirc.  We added the `startedAt` field.